code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1185,6 +1190,89 @@
 		M.confused = min(M.confused + 2, 5)
 	..()
 
+/datum/reagent/carbon_monoxide
+	name = "Carbon Monoxide"
+	description = "A highly dangerous gas for sapients."
+	reagent_state = GAS
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
+	color = "#96898c"
+	var/accumilation
+
+/datum/reagent/carbon_monoxide/on_mob_life(mob/living/carbon/victim)
+	if(holder.has_reagent(/datum/reagent/oxygen))
+		holder.remove_reagent(/datum/reagent/carbon_monoxide, 2*REM)
+		accumilation = accumilation/4
+
+	accumilation += volume
+	switch(accumilation)
+		if(10 to 50)
+			to_chat(src, "<span class='warning'>You feel dizzy.</span>")
+		if(50 to 150)
+			to_chat(victim, "<span class='warning'>[pick("Your head hurts.", "Your head pounds.")]</span>")
+			victim.Dizzy(5)
+		if(150 to 250)
+			to_chat(victim, "<span class='userdanger'>[pick("Your head hurts!", "You feel a burning knife inside your brain!", "A wave of pain fills your head!")]</span>")
+			victim.Stun(10)
+			victim.Dizzy(5)
+			victim.confused = (accumilation/50)
+			victim.gain_trauma(/datum/brain_trauma/mild/expressive_aphasia)
+			victim.gain_trauma(/datum/brain_trauma/mild/muscle_weakness)
```

Comment:
The traumas here should probably be tracked somehow so huffing monoxide doesn't fix unrelated ones

Reply:
There's no way to track it as far as I'm aware, the < 140 is a bandaid